packages/stream_video/lib/src/webrtc/media/camera_constraints.dart

```diff
@@ -52,11 +57,13 @@ class CameraConstraints extends VideoConstraints {
     FacingMode? facingMode,
     String? deviceId,
     double? maxFrameRate,
+    bool? Function()? mirror,
```

Comment:
Why a function here?

Reply:
It's a nullable field so you can't set it to null using copyWith otherwise. How do we handle it in other places?

Reply:
Why not make it non-nullable then?

Reply:
Set it to false by default?

Reply:
Because by default it should be true for front camera and false for back camera. So it has 3 states. True, false, default (null). We could make it an enum